x/rollup/keeper/deposits.go

```diff
@@ -97,11 +97,14 @@ func (k *Keeper) processL1UserDepositTxs(
 // Currently, only finalizeBridgeETH and finalizeBridgeERC20 messages from the L1StandardBridge are recognized for minting tokens
 // on the Cosmos chain. If a message is not recognized, it returns nil and does not error.
 func (k *Keeper) processCrossDomainMessage(ctx sdk.Context, txData []byte) (*sdk.Event, error) { //nolint:gocritic // hugeParam
-	crossDomainMessengerABI, err := abi.JSON(strings.NewReader(opbindings.CrossDomainMessengerMetaData.ABI))
+	crossDomainMessengerABI, err := abi.JSON(strings.NewReader(crossdomainbindings.L2CrossDomainMessengerMetaData.ABI))
 	if err != nil {
 		return nil, fmt.Errorf("failed to parse CrossDomainMessenger ABI: %v", err)
 	}
-	standardBridgeABI, err := abi.JSON(strings.NewReader(opbindings.StandardBridgeMetaData.ABI))
+	// It is a bit lazy to use the L1StandardBridge ABI instead of the L2StandardBridge ABI, but the bindings
+	// package doesn't have the latter. Plus, we know that the functions we want are implemented in the StandardBridge
+	// abstract contract and aren't overridden.
```

Comment:
Good catch here. We could always create a local copy of the bindings for the L2StandardBridge, however I agree with you that it probably isn't necessary.

Reply:
Yeah it can be a future item if we get around to it.